forcepointwebsecurity_connector.py

```diff
@@ -653,13 +655,13 @@ def _handle_lookup_object(self, param):
 
         # Separate each object into their own data result
         # Each object can be 'ip' or 'url'
-        for obj, categories in category_map.iteritems():
+        for obj, categories in list(category_map.items()):
```

Comment:
Typing to `list` is not required over here. Please revert this wherever applicable.